xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1030 @@
+      </para>
+<screen>&prompt.root;<command>systemctl enable --now registry.service</command>
+         </screen>
+     </step>
+    </procedure>
+
+    <para>
+     Now you should have the registries set up and listening on port
+     <literal>5000</literal> on their respective servers.
+    </para>
+    <para>
+     You must now perform the shared update procedure to populate the registry.
+     Refer to: <xref linkend="sec.deploy.scenarios.airgap.update"/>.
+    </para>
+   </sect3>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.container-registry.clients">
+    <title>Client Configuration</title>
+    <para>
+     Using the airgapped mirror works exactly like using a traditional
+     on-premise mirror. You must configure the internal registry mirror in
+     &dashboard;.
+    </para>
+    <procedure>
+     <title>Configuring Registry Mirror</title>
+     <step>
+      <para>
+       Log in to &dashboard; on the &productname; admin node.
+      </para>
+     </step>
+     <step>
+      <para>
+       Navigate to <guimenu>Settings &rarr; Mirrors</guimenu> and create a
+       definition for your internal registry mirror as described in
+       <xref linkend="sec.admin.velum.mirror"/>.
+      </para>
+     </step>
+    </procedure>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.helm-charts">
+   <title>Helm Chart Repository Mirror</title>
+   <para>
+    The helm charts will require images available from a registry mirror. The
+    following steps will aid you in mirroring all images required by the &suse;
+    provided helm charts. To make use of the helm charts, you must complete
+    <xref linkend="sec.deploy.scenarios.airgap.container-registry"/>.
+   </para>
+   <sect3>
+    <title>Mirror Configuration</title>
+    <para>
+     You must have a working container image registry mirror in place inside
+     the air gapped network. This can be any webserver capable of hosting static
+     files. We recommend providing the webserver on your internal mirror server.
+    </para>
+    <procedure>
+     <title>Setting Up Chart Repository Mirror</title>
+     <step>
+      <para>
+       Install and configure a webserver (for example <link xlink:href="https://www.suse.com/documentation/sles-12/book_sle_admin/data/cha_apache2.html">Apache</link>)
+       on your airgapped mirror server.
+      </para>
+     </step>
+     <step>
+      <para>
+       SSH into your mirror server on the <literal>external</literal> network.
+      </para>
+     </step>
+     <step>
+      <para>
+       Download all helm charts from the helm chart repository using
+       <command>helm-mirror</command>.
+      </para>
+<screen>&prompt.user;helm-mirror https://kubernetes-charts.suse.com /chartserver/charts
+      </screen>
+     </step>
+    </procedure>
+   </sect3>
+   <sect3>
+    <title>Client Configuration</title>
+    <para>
+     Add the webserver as a repo to <command>helm</command> on the admin node.
```

Comment:
Maybe you should say something like "go back to the airgap network", the previous step was done inside of the airgap network.

Plus remember what I wrote above about the usage of helm (the client/server architecture)

Reply:
Yes I still need to work this helm distinction in. The other comment is still open because of that.